package.json

```diff
@@ -9,8 +9,16 @@
   },
   "scripts": {
     "test": "cross-env NODE_ENV=test jest --verbose",
-    "lint": "jsonsort src/locale && eslint --fix src --ext .js,.ts,.tsx && stylelint --fix \"src/**/*.scss\"",
-    "lint-check": "eslint src --ext .js,.ts,.tsx --cache --cache-location .eslintcache",
+    "check:eslint": "eslint src --ext .js,.ts,.tsx --cache --cache-location .eslintcache",
+    "lint": "run-p lint:*",
+    "lint:eslint": "eslint src --ext .js,.ts,.tsx",
+    "lint:prettier": "prettier src/**/*.{js,ts,tsx,scss,html} --check",
+    "lint:stylelint": "stylelint \"src/**/*.scss\"",
+    "fix": "run-s fix:eslint fix:json fix:stylelint fix:prettier",
```

Comment:
what's the difference between run-s and run-p? Can this be `run-s fix:*`?

Reply:
run-p is run in parallel, run-s is sequential